classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+
+  def add_student(student)
+    @student.push(student)
```

Comment:
[OPTIONAL]

- Same here as well. Kindly add the getters and setters at the top of the `def initialize` method below the class.

Reply:
@Dipeshtwis I didn't notice, thank you.